Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/groovy
-/* groovylint-disable DuplicateMapLiteral, DuplicateNumberLiteral
-   groovylint-disable DuplicateStringLiteral, NestedBlockDepth, VariableName
-*/
+/* groovylint-disable-next-line LineLength */
+/* groovylint-disable DuplicateMapLiteral, DuplicateNumberLiteral */
+/* groovylint-disable DuplicateStringLiteral, NestedBlockDepth, VariableName */
```

Comment:
Separately I've recently changed to vscode, what plugin are you using to get these warnings as I've not installed any yet and want to ensure I'm using the same as you.